server/game/cards/19-JS/SerBorosBlount.js

```diff
@@ -11,7 +11,7 @@ class SerBorosBlount extends DrawCard {
             gameAction: GameActions.search({
                 title: 'Select a character',
                 topCards: 10,
-                match: { type: 'character', trait: ['Kingsguard', 'Knight'], not: { faction: 'lannister' } },
+                match: { type: 'character', or: { trait: ['Knight'], not: { faction: 'lannister' } }, or: { trait: ['Kingsguard'] } },
```

Comment:
Close; the `or` is relatively new to the matcher, and simply takes in an array of "or matches" to match. For example, the above should look like this:
```suggestion
                match: { type: 'character', or: [{ trait: ['Knight'], not: { faction: 'lannister' } }, {trait: 'Kingsguard'}] },
```
The above `or` is an array containing the element `{ trait: ['Knight'], not: { faction: 'lannister' } }` and the element `{trait: 'Kingsguard'}`

Happy to approve once adjusted :)

Reply:
Hi! Just updated it!